### ISSUE TITLE
[FR]: Example of how to setup multiple `package.json` with `WORKSPACE` macros

### ISSUE THREAD

**Issue by Aghassi** [aspect-build/bazel-examples#181]
### What is the current behavior?

Right now you have to define an `npm_translate_lock` for each `package.json` file in your repo. If you aren't in the space where you can do a pnpm workspace, or you don't want to for some reason, you will end up needing to define an `npm_translate_lock` for every `package.json` in the `WORKSPACE`. This is repetitive and needs three calls per lockfile (which is a pain). It would be nice to have it either:

1. automatically discover all lockfiles and dynamically register them (or have an example of how to do this)
2. Have an example that has two macros in `WORKSPACE`. 1 that defines the `npm_translate_locks` dynamically based off all `pnpm-lock.yaml` in the repo (with an attribute to exclude any, include should be default) and then another macro for loading what is defined and calling the function (this file could be generated maybe?).

Basically we have:

```starlark
# First add the package in this file
load("//bazel/js/pnpm:deps.bzl", "translate_pnpm_lockfiles")

translate_pnpm_lockfiles()

# Then load it in this file and call the fetch logic
load("//bazel/js/pnpm:fetch_pnpm_js_deps.bzl", "fetch_pnpm_js_deps")

fetch_pnpm_js_deps()
```

This would make it much cleaner if there were an example of this. These are order dependent too because `fetch` can only happen after translate lock

### Describe the feature

See above

### Fund our work

- [ ] Sponsor our open source work by donating a [feature bounty](https://opencollective.com/aspect-build/)

**Comment by alexeagle** [comment]
I don't think we should illustrate an example of more than one npm_package_lock per Bazel workspace. That causes duplicate install work and isn't recommended.
